File: 717-1-bit-and-2-bit-characters/717-1-bit-and-2-bit-characters.cpp
class Solution {
public:
    bool isOneBitCharacter(vector<int>& bits) {
        int len = bits.size();
        int pos = 0;
        bool one = true;
        for (int i = 0; i < len; i++) {
            one = true;
            if (bits[i] == 1) {
                i++;
                one = false;
            }
        }
        return one; 
    }
};

File: 717-1-bit-and-2-bit-characters/717-1-bit-and-2-bit-characters.py
class Solution(object):
    def isOneBitCharacter(self, bits):
        """
        :type bits: List[int]
        :rtype: bool
        """
        n= len(bits)
        i=0
        zero=True
        while i<n:
            zero=True
            if bits[i]==1:
                zero =False
                i+=1
            i+=1
        return zero
             